server/common/server.ts

```diff
@@ -17,10 +18,11 @@ export default class ExpressServer {
         const requestLimit = '100kb';
         const sessionSecret = 'secret';
         const root = path.normalize(__dirname + '/../..');
+        tunnel.init();
         app.set('appPath', root + 'client');
-        app.use(bodyParser.json({ limit: requestLimit || '100kb' }));
-        app.use(bodyParser.urlencoded({ extended: true, limit: requestLimit || '100kb' }));
-        app.use(bodyParser.text({ limit: requestLimit || '100kb' }));
+        app.use(bodyParser.json({limit: requestLimit || '100kb'}));
+        app.use(bodyParser.urlencoded({extended: true, limit: requestLimit || '100kb'}));
+        app.use(bodyParser.text({limit: requestLimit || '100kb'}));
```

Comment:
I know that this hasn't changed but some of the CCD api requests return more that 100K of data. How will this setting work with that

Reply:
Yes I agree. Again I didn't explicitly code this part, I only added in the tunnel.init(). However I believe the intention was to abstract these out into config values, but have sensible defaults (the scaffolding tool provided the defaults)